app/src/main/res/values-night/themes.xml

```diff
@@ -1,9 +1,4 @@
 <resources>
-    <style name="Theme.SportiduinoApp" parent="Theme.MaterialComponents.DayNight.DarkActionBar">
```

Comment:
This style now completely copies the normal (day) mode, except for the font size, it is not defined here. It would be possible to add the same font size as in the day mode (but why), but since the color variables are redefined in the `/app/src/main/res/values-night/colors.xml` file, there is nothing left to redefine, so this style block requires deletion.